### PR TITLE
scripts: RocksDB uses 4.12.fb version.

### PR DESCRIPTION
**Pull request by siddontang**
@shenli @qiuyesuifeng

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@qiuyesuifeng PTAL

File: scripts/build_rocksdb.sh [hunk 1]
```diff
@@ -31,14 +31,14 @@ function get_linux_platform {
 
 function install_in_ubuntu {
     echo "building RocksDB in Ubuntu..."
-    if [ ! -d rocksdb-rocksdb-${ROCKSDB_VER} ]; then
+    if [ ! -d rocksdb-${ROCKSDB_VER} ]; then
         ${SUDO} apt-get update 
         ${SUDO} apt-get install -y --no-install-recommends zlib1g-dev libbz2-dev libsnappy-dev libgflags-dev liblz4-dev 
-        curl -L https://github.com/facebook/rocksdb/archive/rocksdb-${ROCKSDB_VER}.tar.gz -o rocksdb.tar.gz 
+        curl -L https://github.com/facebook/rocksdb/archive/$ROCKSDB_VER.tar.gz -o rocksdb.tar.gz
```
Comment (qiuyesuifeng) [inline scripts/build_rocksdb.sh]: How about `${ROCKSDB_VER}`?

Comment (siddontang) [inline scripts/build_rocksdb.sh]: no different.

Comment (siddontang) [inline scripts/build_rocksdb.sh]: but I will change it for unification

**Pull request merged**